app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  def comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I suggest renaming this method `comments_counter` to something like `update_comments_counter` so that there will be a distinction between the methods name and the columns name `:comments_counter`. You can also make use of the `after_save` rails active callback just as you did in the post model.